src/kaleidoscope_internal/LEDModeManager.h

```diff
@@ -23,7 +23,7 @@
 
 #include <stddef.h>
 
-#ifdef KALEIDOSCOPE_VIRTUAL_BUILD
+#if defined(KALEIDOSCOPE_VIRTUAL_BUILD) || defined(ARDUINO_ARCH_STM32)
```

Comment:
This *feels* like it's going to end up being a hassle longer term. But that's not a blocker to merge.